backend/backend.js

```diff
@@ -46,6 +45,12 @@ app.get("/pins", async (req, res) => {
   }
 });
 
+app.get("/pins/:id", async (req, res) => {
```

Comment:
I believe these changes are already merged to main. Can you check to see why these additions are here?

Reply:
Just merged with main and fixed the problem. Thanks David.